File: Exercicios_Em_Python_CV/desafio_007.py
#Faça um programa que leia as duas notas de um aluno, calcule e mostre a média
nota1 = float(input('Digite quanto você tirou na primeira nota:'))
nota2 = float(input('Digite quanto você tirou na segunda nota:'))
media = (nota1+nota2)/2 
print('A média obtida a partir das notas {} e {} é:{:.2f}'.format(nota1, nota2, media))

File: Exercicios_Em_Java_CV/Desafio007.java
package fundamentos;

import java.util.Scanner;

/**
 * Desenvolva um programa que leia as duas notas de um aluno,
 * calculee mostre a sua média
 */
public class Desafio007 {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		System.out.print("Digite a sua primeira nota:");
		float nota1 = scanner.nextFloat();
		scanner.nextLine();
		System.out.println("Digite a sua segunda nota:");
		float nota2 = scanner.nextFloat();
		scanner.nextLine();
		float media = (nota1+nota2)/2;
		System.out.printf("O aluno ficou com média:%.2f", media);
		scanner.close();
		
	}
}
